cmd/mirror/verify.go

```diff
@@ -98,12 +99,22 @@ func getViews(ctx context.Context, dest *database.DB, schema string) (tables []s
 }
 
 func countEntries(ctx context.Context, client *database.DB, table string) (count int) {
+	instanceClause := instanceClause()
+	noInstanceIDColumn := []string{
+		"projections.instances",
+		"projections.system_features",
+		"system.encryption_keys",
+	}
```

Comment:
As the revision of the projections might get increased i would suggest to use the constant table names here.

The table names can be found in the projection package (import: `"github.com/zitadel/zitadel/internal/query/projection"`)

constant name for instance: `projection.InstanceProjectionTable`

Reply:
Using the constants now!